pages/views/frontend.py

```diff
@@ -56,63 +56,32 @@ def frontend(request):
         request.session.set_expiry(0)
         
         # grab client cert form the request create user hash, ignore if no cert info is found in request
-        
         try:
             certInfo = getCert(request)
+            cftsUser = getOrCreateUser(request, certInfo)
+
+            if cftsUser == None or not request.user.is_authenticated:
+                return redirect("/login")
+            elif cftsUser.update_info == True:
```

Comment:
-     replace message.info() in the cftsUser.update_info check before the redirect

Reply:
Done